### PR TITLE
py-llfuse: update to 1.5.0

### PR DESCRIPTION
**Pull request by newtonne**
Also adds a python312 version which fixes the borgbackup +fuse variant which broke when that was updated to use python312: de2b363

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 14.2.1 23C71 arm64
Command Line Tools 15.1.0.0.1.1700200546

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: python/py-llfuse/Portfile [hunk 1]
```diff
@@ -5,11 +5,11 @@ PortGroup           python 1.0
 PortGroup           fuse 1.0
 
 name                py-llfuse
-version             1.3.8
+version             1.5.0
 revision            1
```
Comment (reneeotten) [inline python/py-llfuse/Portfile:9]: the "revision" field should be reset to "0" when updating the "version"

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: python/py-llfuse/Portfile (hunk 1 above)
Comment (reneeotten) [reply python/py-llfuse/Portfile:9]: when you nake that change also please add the following line to the body of the commit message:

Closes: https://trac.macports.org/ticket/69071

Comment (newtonne) [reply python/py-llfuse/Portfile]: Thanks. That's all done 👍

**Comment by reneeotten** [comment]
For future reference, please squash your commits next time.

**Pull request merged**